assets/README.md

```diff
@@ -60,3 +60,62 @@ run:
 ```bash
 npm run test:watch
 ```
+
+## Run a Local Version on a Mobile Device
+Disconnect from Virtual Private Network (VPN) or USGS network.
+
+Install a 'tunneling' application (LocalTunnel is an example (this is not an endorsement)).
+
+Start the tunneling application; make a tunnel for each server
+
+```bash 
+An example using LocalTunnel 
+(each command is in a seprate terminal, since each will be a running process)
+lt --port 5050 (for the Flask server)
+lt --port 9000 (for the Static Assets server)
```

Comment:
I think if you set CONTAINER_RUN to True you shouldn't need to run the static assets server separately. What WhiteNoise does is enables the Flask server to serve out the static. So I think this may be unnecessary.

Reply:
That would make things simpler. Testing now . . .